WcaOnRails/app/webpacker/components/Media/MediaTable.jsx

```diff
@@ -0,0 +1,257 @@
+        search
+        selection
+        value={selectedRegion}
+        text={selectedRegion}
+        onChange={handleSelectRegion}
+      >
+        <DropdownMenu>
+          <DropdownItem
+            key={"all"}
+            onClick={handleSelectRegion}
+            value={"All Regions"}
+          >
+            {"All Regions"}
+          </DropdownItem>
+          <DropdownHeader>Continent</DropdownHeader>
+          {continentOptions.map((continent) => (
+            <DropdownItem
+              key={continent.id}
+              onClick={handleSelectRegion}
+              value={continent.text}
+            >
+              {continent.text}
+            </DropdownItem>
+          ))}
+          <DropdownDivider />
+          <DropdownHeader>Region</DropdownHeader>
+          {countryOptions.map((country) => (
+            <DropdownItem
+              key={country.key}
+              onClick={handleSelectRegion}
+              value={country.text}
+            >
+              {country.text}
+            </DropdownItem>
+          ))}
+        </DropdownMenu>
+      </Dropdown>
+
+      <Table sortable celled fixed>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell
+              sorted={sortColumn === 'timestampSubmitted' ? sortDirection : null}
+              onClick={handleSort('timestampSubmitted')}
+            >
+              Submission Date
+            </Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={sortColumn === 'timestampDecided' ? sortDirection : null}
+              onClick={handleSort('timestampDecided')}
+            >
+              Competition Date
+            </Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={sortColumn === 'competetion' ? sortDirection : null}
+              onClick={handleSort('competetion')}
+            >{I18n.t('activerecord.attributes.competition_medium.competitionId')}</Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={sortColumn === 'location' ? sortDirection : null}
+              onClick={handleSort('location')}
+            >{I18n.t('media.media_table.location')}</Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={sortColumn === 'type' ? sortDirection : null}
+              onClick={handleSort('type')}
+            >{I18n.t('activerecord.attributes.competition_medium.type')}</Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={sortColumn === 'link' ? sortDirection : null}
+              onClick={handleSort('link')}>{I18n.t('activerecord.attributes.competition_medium.uri')}</Table.HeaderCell>
+            {isValidate && <Table.HeaderCell></Table.HeaderCell>}
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+
+          {mediaCombined?.map((media_row) => (
+            <Table.Row>
+              <Table.Cell>
+                {moment.utc(media_row.timestampSubmitted).format('MMMM DD, YYYY HH:mm UTC')}
```

Comment:
I think we prefer to use luxon over moment? Someone correct me If I'm wrong.

Reply:
Done